tuplex/core/include/physical/PhysicalStage.h

```diff
@@ -44,16 +48,23 @@ namespace tuplex {
         std::vector<PhysicalStage*> _predecessors;
         int64_t _number;
         std::unordered_map<std::tuple<int64_t, ExceptionCode>, size_t> _ecounts; //! exception counts for this stage.
+        std::vector<LogicalOperator*> _opids;
```

Comment:
the field is called _opids, yet it stores LogicalOperator* objects. operator IDs are size_t though. Would rename to _operators maybe? Also, what is required here? IDs or the full objects?

Reply:
Renamed to _operators. Full objects are required.